example/res/layout/activity_payment_multiline.xml

```diff
@@ -31,12 +31,14 @@
     <TextView android:id="@+id/source_list_title"
               android:text="@string/paymentMethods"
               android:layout_marginTop="12dp"
-              style="@style/Header.Light" />
+              style="@style/Header.Light"
+        />
 
-    <ListView
-        android:id="@+id/source_list"
+    <android.support.v7.widget.RecyclerView
```

Comment:
Actually, the masked widget can't (at the moment) be filled with Token objects. You can use cards from the CardInputWidget, but not tokens, as the API is using Customer objects or Source objects directly.